container/examples/container-iframe/microfrontend.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title></title>
+    <meta charset="utf-8" />
+  </head>
+
+  <body >
```

Comment:
```suggestion
  <body style="border: solid blue 2px;">
```

To make it visually easy to see where the container iframe stands.